OptionsCreator.py

```diff
@@ -560,8 +571,12 @@ def create_options_panel(self, world_button: WorldButton):
                 groups[group].append((name, option))
 
             for group, options in groups.items():
+                options = [(name, option) for name, option in options
+                           if name and option is not Removed and
+                           option.visibility & Visibility.simple_ui]
```

Comment:
I think I would prefer this, it doesn't really make sense to me to reject elements in a comprehension based on a condition that only involves outside static data. This is longer, but I think it'd be clearer

```suggestion
                if not option.visibility & Visibility.simple_ui:
                    continue
                if option is Removed:
                    continue
                options = [(name, option) for name, option in options if name]
```

Reply:
Also is "is not Removed" the correct way to do this? There is one Stardew Valley option that actually *subclasses* Removed, so I think it should be isinstance, or alternatively Stardew Valley is wrong for doing that

Reply:
(Although for what it's worth, `Removed` already has Visibility.none so technically it's not even needed, but I think the explicitness is fine)

Reply:
The suggestion here though is wrong though. `name` and `option` are both being iterated over in the comprehension, so both are relevant to the comprehension (ie non-static).

Reply:
I'm blind